src/components/ListView.js

```diff
@@ -7,12 +7,48 @@ const ListView = ({ shoppingList, loading, error }) => {
   const [shoppingListEmpty, setShoppingListEmpty] = useState(true);
   const [length, setLength] = useState(0);
   const [value, setValue] = useState('');
+  const [sortedList, setSortedList] = useState([]);
+  //indicator for urgency status of item
+  let purchaseIndex;
 
   const handleChange = (e) => setValue(e.target.value);
 
+  //assign value to purchaseIndex based on these parameters
+  const checkIndex = (lastPurchase, nextPurchase, timesPurchased) => {
+    //check interval between current date and date of last purchase
+    //set index as inactive if item has been purchased only once or less
+    //or if interval is >= 2 times the projected estimate
+    const interval = new Date().getTime() - lastPurchase;
+    if (timesPurchased <= 1 || interval >= 2 * nextPurchase * 86400000) {
```

Comment:
What does 86400000 symbolize here? Generally it's an anti-pattern to use so called "[Magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=The%20term%20magic%20number%20or,numbers%20directly%20in%20source%20code.&text=Replacing%20all%20significant%20magic%20numbers,to%20read%2C%20understand%20and%20maintain.)" like this :slightly_smiling_face: . It would be better if you assigned the number to a descriptive variable so it's easier to understand what's going on.

Reply:
The number is for converting days into milliseconds (24*60*60*1000).
I'll effect the change now, thank you.

Reply:
Hi @faysvas, I've corrected it, thank you